src/main/java/com/scalar/db/storage/jdbc/query/MergeIntoQuery.java

```diff
@@ -43,16 +42,21 @@ protected String sql() {
     List<String> enclosedValueNames =
         values.keySet().stream().map(n -> enclose(n, rdbEngine)).collect(Collectors.toList());
 
-    return "MERGE INTO "
-        + enclosedFullTableName(schema, table, rdbEngine)
-        + " t1 USING (SELECT "
-        + makeUsingSelectSqlString(enclosedKeyNames)
-        + " FROM DUAL) t2 ON ("
-        + makePrimaryKeyConditionsSqlString(enclosedKeyNames)
-        + ") WHEN MATCHED THEN UPDATE SET "
-        + makeUpdateSetSqlString(enclosedValueNames)
-        + " WHEN NOT MATCHED THEN INSERT "
-        + makeInsertSqlString(enclosedKeyNames, enclosedValueNames);
+    StringBuilder sql = new StringBuilder();
```

Comment:
Is it changed o use StringBuilder without changing what it does?
If so, is there any particular reason? (just for consistency?)

Reply:
Thank you for asking!

I changed what it did actually. I changed the behavior in the case that `values` is empty as follows:
https://github.com/scalar-labs/scalardb/pull/180/files#diff-809f5679795529b62ebe4e9f3370fbbb63553a5eb75db56050cee6633e82d2a4R53-R56

Reply:
Oh, I missed it. Thank you! 👍